mmv1/products/looker/Instance.yaml

```diff
@@ -69,9 +69,9 @@ examples:
       client_id: 'my-client-id'
       client_secret: 'my-client-secret'
     test_vars_overrides:
-      address_name: 'acctest.BootstrapSharedTestGlobalAddress(t, "looker-vpc-network-1", acctest.AddressWithPrefixLength(20))'
+      address_name: 'acctest.BootstrapSharedTestGlobalAddress(t, "looker-vpc-network-1")'
```

Comment:
We may need to change the name `looker-vpc-network-1` to something else to test if this works, as based on how we set up the bootstrapped resource, they won't get recreated without the name changed (we'll read the resource first and only create a new one if no existing one is found)

Reply:
@shuyama1 Okay, done!